package.json

```diff
@@ -6,14 +6,16 @@
     "@itsjoekent/geokdbush": "^1.1.1",
     "aws-sdk": "^2.395.0",
     "geokdbush": "^1.1.0",
-    "kdbush": "^3.0.0"
+    "kdbush": "^3.0.0",
```

Comment:
Just leaving a note here that eventually the following packages should be uninstalled,

- `"@itsjoekent/geokdbush"`
- `geokdbush`
- `kdbush`

These packages were used for calculating distance between lat/lon coords but Mongo will do that for us now.